README.md

```diff
@@ -12,14 +12,28 @@ catkin_make
 ```
 
 ## Running
-If you have a .world file, place it in `/worlds` and modify the `ttb3_velodyne.launch` file to load your world.
+If you have a .world file, place it in `/worlds` and modify the respective `.launch` file to load your world.
 
 ```xml
 <arg name="world_name" value="$(find turtlebot3_ramp_sim)/worlds/your_world_here.world"/>
 ```
+
 Launch the simulation
+
 ```ros
-roslaunch turtlebot3_ramp_sim ttb3_velodyne.launch
+roslaunch turtlebot3_ramp_sim ttb3_(sensor name).launch
```

Comment:
you can make this something like:
```
export PERCEP_SENSOR="hokuyo"
roslaunch turtlebot3_ramp_sim ttb3_$PERCEP_SENSOR.launch
```
or you can also make a common launch file and add arguments inside the launch file to trigger various other launch files

Reply:
Combined into a single launch file, updated README.